### PR TITLE
moved CD to Travis

### PR DESCRIPTION
**Pull request by rporres**
Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -122,6 +124,14 @@ jobs:
       script:
         - PKG_OS=linux make -f Makefile.dummy build
         - DOCKER_PUSH_LATEST=true make -f Makefile.dummy docker-push
+    - name: 'push staging deploy image to DockerHub'
+      stage: deploy staging
+      script:
+        - make dependencies
+        - PKG_OS=linux DOCKER_OS=linux DOCKER_ARCH=amd64 make packages
+        - make docker-push
+        - echo HELM_RELEASE=lookout HELM_CHART=lookout K8S_NAMESPACE=lookout HELM_ARGS="--namespace lookout --tiller-namespace lookout --repo https://src-d.github.io/charts/lookout/ --set image.tag=dev-${TRAVIS_COMMIT:0:7} --version 0.1.0 -f .helm-staging.yml" make deploy
```
Comment (dpordomingo) [inline .travis.yml]: I guess `echo` is a leftover

File: .travis.yml [hunk 2]
```diff
@@ -35,6 +35,8 @@ stages:
     if: tag IS present AND NOT tag =~ /^dummy/
   - name: release dummy analyzer
     if: tag IS present AND tag =~ /^dummy/
+  - name: deploy staging
+    if: branch = move-cd-to-travis AND type = push
```
Comment (dpordomingo) [inline .travis.yml]: I guess this should be `master` as in drone, shouldn't it?

**Comment by rporres** [comment]
Apologies, @dpordomingo . I should have tagged this as a [WIP] as I've just done now

**Comment by rporres** [comment]
Ready to be reviewed

File: .travis.yml [hunk 3]
```diff
@@ -122,6 +124,14 @@ jobs:
       script:
         - PKG_OS=linux make -f Makefile.dummy build
         - DOCKER_PUSH_LATEST=true make -f Makefile.dummy docker-push
+    - name: 'push staging deploy image to DockerHub'
+      stage: deploy staging
+      script:
+        - make dependencies
+        - PKG_OS=linux DOCKER_OS=linux DOCKER_ARCH=amd64 make packages
+        - make docker-push
+        - HELM_RELEASE=lookout HELM_CHART=lookout K8S_NAMESPACE=lookout HELM_ARGS="--namespace lookout --tiller-namespace lookout --repo https://src-d.github.io/charts/lookout/ --set image.tag=dev-${TRAVIS_COMMIT:0:7} --version 0.9.1 -f .helm-staging.yml" make deploy
```
Comment (meyskens) [inline .travis.yml]:
```suggestion
        - HELM_RELEASE=lookout HELM_CHART=lookout K8S_NAMESPACE=lookout HELM_ARGS="--namespace lookout --tiller-namespace lookout --repo https://src-d.github.io/charts/stable/ --set image.tag=dev-${TRAVIS_COMMIT:0:7} --version 0.9.1 -f .helm-staging.yml" make deploy
```

**Comment by dpordomingo** [comment]
Adding @src-d/applications to reviewers list, to accomplish our protocol to merge.
Ideally, it should be also approved by the maintainer, but since @carlosms is no longer in apps, I'd say that any other approval would be enough.

**Pull request merged**